ivy/functional/backends/numpy/elementwise.py

```diff
@@ -12,8 +12,14 @@
 
 @_scalar_output_to_0d_array
 def abs(
-    x: Union[float, np.ndarray], /, *, out: Optional[np.ndarray] = None
+    x: Union[float, np.ndarray],
+    /,
+    *,
+    where: Union[bool, np.ndarray] = True,
+    out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
+    if where & np.absolute(x, out=out):
```

Comment:
`where` in this case wouldn't need to be in the `if` statement, `where` is supposed to indicate which elements we would convert to their absolute values rather than whether we would apply the function to the array or not